testregression/flakytests.md

```diff
@@ -1,146 +0,0 @@
-Flaky test
-================
-
-
```

Comment:
Awesome improvement turning this file into a `.ipynb` to make possible the inline code without duplication.